.github/workflows/upstream-tests.yaml

```
@@ -181,20 +201,29 @@ jobs:
       - name: Run profiler image
         timeout-minutes: 10
         run: docker run --network none -v /dev/hugepages-1G:/dev/hugepages-1G --device /dev/tenstorrent ${{ needs.get-image-tags.outputs.wh-6u-profiler-image-tag }}
-  test-bh-llmbox-image:
+  test-bh-multicard-image:
     needs:
       - get-image-tags
       - build-images
+    strategy:
+      matrix:
```

```suggestion
    strategy:
      fail-fast: false
      matrix:
```

This should prevent the parallel jobs from cancelling

Should we also comment out the deskbox and rackbox runs until the fabric changes are in?